tests/llama_stack/conftest.py

```diff
@@ -85,11 +91,52 @@ def llama_stack_server_config(
             "port": 8321,
         },
         "distribution": {"name": "rh-dev"},
-        "storage": {
-            "size": "20Gi",
-        },
     }
 
+    if params.get("llama_stack_storage_size"):
+        storage_size = params.get("llama_stack_storage_size")
+        server_config["storage"] = {"size": storage_size}
+
+    return server_config
+
+
+@pytest.fixture(scope="class")
+def llama_stack_user_config_configmap(
```

Comment:
This fixture should be deleted. In a follow up PR I can add a new version using a costant, as suggested in the meeting